aws/configure

```diff
@@ -160,6 +160,8 @@ echo "${ZONE_ID:-not found}"
 
 if $DRY_RUN; then
   echo "* Dry run mode detected: skipping Route53 modification"
+elif test -z "$ZONE_ID"; then
```

Comment:
This code now always skips DNS zone creation if it does not exist as far as I understand, which is not correct. 

We have the following convention for `gcp` and `azure` right now and have not introduced a better approach yet (@akranga to confirm): 
- if the params file contains `dns.domain` parameter, then we create a DNS zone for the given domain name
- if the params file does not contain `dns.domain` parameter, then we do not create a DNS zone and warn a user that the domain will not be created.

We need to do it in a similar way for AWS.
You can use the `params` utility to check if the parameter is present in one of the hub files: https://github.com/epam/hub-extensions/blob/develop/bin/params